app/views/job_runs/show.html.erb

```diff
@@ -5,16 +5,16 @@
   </div>
 </div>
 <div class="row p-1 pl-5">
+  <div class="col-6">
+    <%= render partial: 'settings', locals: { batch_context: @job_run.batch_context } %>
+  </div>
   <div class="col-6">
     <%= render partial: 'details' %>
   </div>
 </div>
 <% if @job_run.report_ready? %>
     <turbo-frame id="discovery-report-frame" src="<%= discovery_report_summary_job_run_path(@job_run) %>"></turbo-frame>
 <% end %>
-<% if @job_run.progress_log_file_exists? %>
-    <turbo-frame id="progress-log-frame" src="<%= progress_log_job_run_path(@job_run) %>"></turbo-frame>
-<% end %>
 <div class="row mt-5">
```

Comment:
this should NOT be in the job run view anymore ... it was removed in a previous PR (https://github.com/sul-dlss/pre-assembly/pull/1269) but was inadvertently re-added by a munged rebase for accessibility work (https://github.com/sul-dlss/pre-assembly/pull/1265)